gix-fsck/tests/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
Done, and thank you! I was a bit confused because by attempt at partial cloning using the existing test tooling was causing my tests to fail, but that ended up being due to `uploadpack.allowFilter`not being set in persistent git config. See below for details on that...